sdk/crystal/src/config_snapshots.cr

```diff
@@ -47,5 +48,19 @@ module StorageManager
         StorageManager.error_response(response)
       end
     end
+
+    def self.get_config_snapshot(client : Client, snap_name : String)
+      url = "#{client.url}/api/v1/config-snapshots/#{snap_name}"
```

Comment:
Change this to `url = "#{@client.url}/api/v1/config-snapshots/#{@snap_name}"`

Reply:
added @